datasource/src/test/scala/quasar/physical/mongo/MongoScalarStagesInterpreterSpec.scala

```diff
@@ -82,7 +99,8 @@ class MongoScalarStagesInterpreterSpec
         (CPathField("a1"), (CPathField("a0"), Nil)),
         (CPathField("b1"), (CPathField("b0"), Nil)))
 
-      input must cartesianInto(targets)(expected)
+      val actual = interpret(ScalarStages(IdStatus.ExcludeId, List(ScalarStage.Cartesian(targets))), input, (x => x))
+      actual must bestSemanticEqual(expected)
     }
 
     "nested pivoting doesn't produce unnecessary empty fields" in {
```

Comment:
This test is `cart-7` in `ScalarStageSpec`. Can we delete it here?

Reply:
Absolutely!